app/controllers/reviewables_controller.rb

```diff
@@ -179,7 +179,7 @@ def update
   end
 
   def perform
-    args = { version: params[:version].to_i }
+    args = { version: params[:version].to_i, reject_reason: params[:reject_reason], send_email: params[:send_email] == "true" }
```

Comment:
I would prefer if these arguments were only added if they exist and that is the correct reviewable type.